Fakebook.Posts/Fakebook.Posts.Domain/Models/Post.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Fakebook.Posts.Domain {
+    public class Post {
+
+        public int Id { get; set; }
+        public string Content { get; set; }
+        public string Picture { get; set; }
+        public DateTime CreatedAt { get; set; }
+        public int UserId { get; set; }
+        
+        public ICollection<Comment> Comments { get; set; }
+
+        public Post() {
+            Comments = new List<Comment>();
+        }
+    }
+}
```

Comment:
just skeleton code here, we'll consider the domain model in other PRs

Reply:
Noted, removed the properties inside the post and comment domain models for now